taskchampion/src/task/task.rs

```diff
@@ -225,6 +241,20 @@ impl<'r> TaskMut<'r> {
         self.set_string(format!("tag.{}", tag), None)
     }
 
+    /// Add a new annotation.  Note that annotations with the same entry time
+    /// will overwrite one another.
```

Comment:
This seems a little odd, but, presumably just constrained by TW?

Reply:
Yeah, and in practice it's hard to make two annotations in one second :)

Reply:
Only vaguely plausible thing I can think of is `ta 1 annotate Did thing; ta +active annotate Another thing in batch` but seems quite esoteric, doubly so if that's how TW has worked for hwoever long!